com.bktus.gpgfrontend.yaml

```diff
@@ -0,0 +1,96 @@
+app-id: com.bktus.gpgfrontend
```

Comment:
Do you own/control the domain here `bktus.com`?

Reply:
yes

Reply:
You can verify it by visiting https://www.gpgfrontend.bktus.com/